atlasexec/atlas_test.go

```diff
@@ -131,7 +131,7 @@ func TestMigrateLint(t *testing.T) {
 			Latest: 1,
 			Writer: &buf,
 		})
-		require.NoError(t, err)
+		require.ErrorContains(t, err, "atlas command exited with 1")
```

Comment:
This is suspicions. No error before -> now there is an error. Are you sure this is correct?

Reply:
Yes, there was a branch in the code that was hiding it (inside runCommand)

We don't want to expose it as an error in MigrateLint for backwards compatibility, but MigrateLintError is new and it's the only way we can tell that the lint failed when running in CI